internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -889,12 +902,16 @@ public CompletableFuture<PubSubProduceResult> put(
     byte[] serializedKey = keySerializer.serialize(topicName, key);
     byte[] serializedValue = valueSerializer.serialize(topicName, value);
     int partition = getPartition(serializedKey);
-
     int replicationMetadataPayloadSize = putMetadata == null ? 0 : putMetadata.getSerializedSize();
     isChunkingFlagInvoked = true;
-    if (serializedKey.length + serializedValue.length
-        + replicationMetadataPayloadSize > maxSizeForUserPayloadPerMessageInBytes) {
-      if (isChunkingEnabled) {
+
+    /**
+     * {@link RecordTooLargeException} will be thrown unless the record size fits within one of the following categories:
+     * Records without Chunking < ~1MB < Records with Chunking < MAX_RECORD_SIZE_BYTES
+     */
+    int veniceRecordSize = serializedKey.length + serializedValue.length + replicationMetadataPayloadSize;
+    if (isChunkingNeededForRecord(veniceRecordSize) /* ~1MB default */) {
+      if (isChunkingEnabled && !isRecordTooLarge(veniceRecordSize) /* 10MB default */) {
```

Comment:
We don't use inline `/* */` comment syntax. To keep consistency, can we replace it with `//` comment syntax at the end of the line?

Reply:
Done.